### PR TITLE
DMC: Bladewing, Deathless Tyrant + 6 cards

### PR DESCRIPTION
**Pull request by Suthro**
DMC: Add scripts for 7 new DMC cards

- Bladewing, Deathless Tyrant
- Cadric, Soul Kindler
- Iridian Maelstrom
- Jenson Carthalion, Druid Exile
- Primeval Spawn
- Shanid, Sleepers' Scourge
- The Peregrine Dynamo

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/bladewing_deathless_tyrant.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Bladewing, Deathless Tyrant
+ManaCost:5 B R
+Types:Legendary Creature Dragon Skeleton
+PT:6/6
+K:Flying
+K:Haste
+T:Mode$ DamageDone | ValidSource$ Card.Self | ValidTarget$ Player,Planeswalker | CombatDamage$ True | Execute$ TrigToken | TriggerZones$ Battlefield | TriggerDescription$ Whenever CARDNAME deals combat damage to a player or planeswalker, for each creature card in your graveyard, create a 2/2 black Zombie Knight creature token with menace.
+SVar:TrigToken:DB$ Token | TokenScript$ b_2_2_zombie_knight_menace | TokenAmount$ X | TokenOwner$ You
+SVar:X:Count$TypeInYourYard.Creature
+DeckHas:Ability$Graveyard & Type$Zombie|Knight
```
Comment (Agetian) [inline forge-gui/res/cardsfolder/upcoming/bladewing_deathless_tyrant.txt:10]: DeckHas:Ability$Token & Type$Zombie|Knight

Maybe:
DeckHints:Ability$Graveyard

File: forge-gui/res/cardsfolder/upcoming/cadric_soul_kindler.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+Name:Cadric, Soul Kindler
+ManaCost:2 R W
+Types:Legendary Creature Dwarf Wizard
+PT:4/3
+S:Mode$ IgnoreLegendRule | ValidCard$ Permanent.token+YouCtrl | Description$ The "legend rule" doesn't apply to tokens you control.
+T:Mode$ ChangesZone | ValidCard$ Permanent.nonToken+Legendary+Other+YouCtrl | Origin$ Any | Destination$ Battlefield | TriggerZones$ Battlefield | Execute$ TrigCopy | TriggerDescription$ Whenever another nontoken legendary permanent enters the battlefield under your control, you may pay {1}. If you do, create a token that's a copy of it. That token gains haste. Sacrifice it at the beginning of the next end step.
+SVar:TrigCopy:AB$ CopyPermanent | Cost$ 1 | Defined$ TriggeredCard | PumpKeywords$ Haste | AtEOT$ Sacrifice
+SVar:BuffedBy:Legendary
+DeckNeeds:Type$Legendary
+Oracle:The "legend rule" doesn't apply to tokens you control.\nWhenever another nontoken legendary permanent enters the battlefield under your control, you may pay {1}. If you do, create a token that's a copy of it. That token gains haste. Sacrifice it at the beginning of the next end step.
```
Comment (Agetian) [inline forge-gui/res/cardsfolder/upcoming/cadric_soul_kindler.txt:10]: DeckHas:Ability$Token

File: forge-gui/res/cardsfolder/upcoming/bladewing_deathless_tyrant.txt (hunk 1 above)
Comment (Suthro) [reply forge-gui/res/cardsfolder/upcoming/bladewing_deathless_tyrant.txt]: Reworked DeckHas and DeckHints flags as recommended

File: forge-gui/res/cardsfolder/upcoming/cadric_soul_kindler.txt (hunk 2 above)
Comment (Suthro) [reply forge-gui/res/cardsfolder/upcoming/cadric_soul_kindler.txt:11]: Added DeckHas token flag to script

**Review by Agetian — APPROVED** [review]

**Pull request merged**